apps/passport-client/components/screens/ProveScreen/GenericProveSection.tsx

```diff
@@ -33,7 +32,7 @@ export function GenericProveSection<T extends PCDPackage = PCDPackage>({
     pendingPCD: PendingPCD | undefined
   ) => void;
 }) {
-  const rollbar = useRollbar();
+  const rollbar = undefined; // useRollbar();
```

Comment:
Not having a rollbar api key will cause this code to fail for OSS devs. I tried to set `rollbar` conditionally, but something about react hooks prevented me from doing that.